tests/foreman/cli/test_user.py

```diff
@@ -133,6 +133,23 @@ def test_positive_CRUD(self):
         with pytest.raises(CLIReturnCodeError):
             User.info({'login': user['login']})
 
+    @pytest.mark.tier1
+    def test_positive_user_create_with_latin_email(self):
+        """Create user with latin email id
+
+        :id: 262a12c2-fd4d-481c-8485-fe70a34ca7b7
+
+        :expectedresults: User should be created with the latin email id
+
+        :CaseImportance: High
+
+        :BZ: 1204667
+        """
+        email = f'{gen_string("latin1")}@example.com'
```

Comment:
There is something to consider with the behavior of `test_positive_CRUD` function above.

It's randomly picking from the datafactory, for the email. This is just a list of emails, defined here:
https://github.com/SatelliteQE/robottelo/blob/e9b0b4df781ce6a8ddb2051a221365cd6fc9eb3e/robottelo/datafactory.py#L295

If latin emails are supported, this use case can and should be covered by adding such an email to the list in the datafactory.

Picking a random email address within the test function is problematic, if you ask me, because you get random test coverage - and this BZ could easily could have gone unnoticed even with this coverage, with latin never getting picked in any of the snap testruns.

I would recommend you modify the datafactory function, and include a latin email.  Double check the other testcases where this list is used, and verify they pass as well.

Then, considering the increase in test execution time, actually parametrize `test_positive_CRUD` with the list of email types.  There's an enhancement that comes with this, and that's not actually randomizing the email itself as the value, but parametrizing the type of email - changing the datafactory list to a dictionary.

I don't want to blow the scope up on your PR here, but there's no need for this BZ to have unique test function coverage within the framework.

Reply:
Thanks for the suggestion Mike.
I have parametrized the test  with email types and kept the datafactory list as the list itself
`changing the datafactory list to a dictionary` might  impact other tests which are treating it as a list across robottelo